boards/remote-reva/include/periph_conf.h

```diff
@@ -99,6 +99,24 @@ static const periph_spi_conf_t spi_config[SPI_NUMOF] = {
 /** @} */
 
 /**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (3)
+#define ADC1_PIN            5
+#define ADC2_PIN            4
+#define ADC3_PIN            2
+
+static const uint8_t periph_adc_map[ADC_NUMOF] = {
+  ADC1_PIN,
+  ADC2_PIN,
+  ADC3_PIN,
```

Comment:
I just tested on a remote-reva but it seems ADC_3 does not work correctly or is at least calibrated completely wrong. Applying 0V level to this pin results in `95`, applying 3,3V results in `1200`. In contrast to that, the other channels sample `0` and `3276` as expected.